SPECS/wsproxy.spec

```diff
@@ -1,15 +1,12 @@
+%global package_speccommit 1d30e9b477299216e788d2f9156528c5bca66424
+%global package_srccommit v1.12.0
 Summary: Websockets proxy for VNC traffic
 Name:    wsproxy
 Version: 1.12.0
-Release: 5%{?dist}
-License: LGPL+linking exception
+Release: 11%{?xsrel}%{?dist}
+license: LGPL-2.0-only WITH OCaml-LGPL-linking-exception
 URL:     https://github.com/xapi-project/wsproxy
-
-Source0: https://code.citrite.net/rest/archive/latest/projects/XSU/repos/wsproxy/archive?at=v1.12.0&format=tar.gz&prefix=wsproxy-1.12.0#/wsproxy-1.12.0.tar.gz
-
-
-Provides: gitsha(https://code.citrite.net/rest/archive/latest/projects/XSU/repos/wsproxy/archive?at=v1.12.0&format=tar.gz&prefix=wsproxy-1.12.0#/wsproxy-1.12.0.tar.gz) = 6e80d9d69b8a460ef3da45adf524ea1146545e1c
-
+Source0: wsproxy-1.12.0.tar.gz
```

Comment:
Isn't this file missing ?

Reply:
My bad, I was not on the good branch. The file is here in the 8.2 branch

Reply:
No it isn't missing. Sources aren't updated on this one. Just the spec file to bump the release because we need it for the build chain.